ban-importer/src/ban-fetcher.js

```diff
@@ -122,8 +122,9 @@ export default class BanFetcher {
             expires: expires,
             expired: !(ban.attributes.expires === null || expires.getTime() > Date.now()),
 
-            reason: classifyBanReason(`${ban.attributes.reason} ${ban.attributes.note}`),
-            rawReason: `${ban.attributes.reason} ${ban.attributes.note}`,
+            reason: classifyBanReason(`${ban.attributes.reason}`,`${ban.attributes.note}`),
+            rawReason: `${ban.attributes.reason}`,
+            rawNotes: `${ban.attributes.note}`,
```

Comment:
As above.

Reply:
fixed in https://github.com/CommunityBanList/Communitybanlist/pull/11/commits/40c517447dfe396de588d13c5dd41cd70efc04b9